### PR TITLE
Webkul-ME-60 - Removing balance vendor id and adding module enable co…

### PR DESCRIPTION
**Pull request by dhiraj-toptal**
…ndition

### CODE REVIEW

**Review by ynaxonBalance — DISMISSED** [review]

**Comment by ynaxonBalance** [comment]
@dhiraj-toptal what about removing the assign product to vendor?

File: Ui/DataProvider/Product/Form/Modifier/AssignVendor.php [hunk 1]
```diff
@@ -51,27 +63,31 @@ public function modifyData(array $data)
...
-                'assign_vendor' => [
-                    'arguments' => [
-                        'data' => [
-                            'config' => [
-                                'label' => __('Assign Product to Vendor'),
-                                'componentType' => Fieldset::NAME,
-                                'dataScope' => 'data.product.assign_vendor',
-                                'collapsible' => false,
-                                'sortOrder' => 5,
+        if (!($this->isModuleEnable())) {
```
Comment (dhiraj-toptal) [inline Ui/DataProvider/Product/Form/Modifier/AssignVendor.php]: This condition checks if Webkul module is enable or not. 
If its enable we remove the "Assign Product to Vendor" dropdown.
@ynaxonBalance

**Comment by dhiraj-toptal** [comment]
> @dhiraj-toptal what about removing the assign product to vendor?

I have added a comment to code @ynaxonBalance

File: Ui/DataProvider/Product/Form/Modifier/AssignVendor.php [hunk 2]
```diff
@@ -149,4 +165,12 @@ public function getAllSellerCollectionObj()
         }
         return $options;
     }
+
+    /**
+     * @return mixed
+     */
+    public function isModuleEnable()
```
Comment (ynaxonBalance) [inline Ui/DataProvider/Product/Form/Modifier/AssignVendor.php]: rename to `isWebkulEnabled`

Comment (dhiraj-toptal) [reply Ui/DataProvider/Product/Form/Modifier/AssignVendor.php]: Renamed the function, @ynaxonBalance

**Review by ynaxonBalance — APPROVED** [review]

**Pull request merged**